test/utils.js

```diff
@@ -0,0 +1,76 @@
+const logger = require('../utils/logger');
+const has = require('has');
+
+class Query {
+  constructor (db) {
+    this.cur = 0;
+    this.db = db;
+  }
+
+  cursor (value) {
+    this.cursor = value;
+    return this;
+  }
+
+  objects () {
+    return this;
+  }
+
+  execute () {
+    let self = this;
+    return new Promise((resolve, reject) => {
+      let keys = Object.keys(self.db.data);
+      let nextCursor = self.cur + 100;
```

Comment:
@drewbo I picked `100` because it's the default increment in the tile38 SCAN queries. Since this is just a mock, what do you suggest?